src/Features/CSharp/Portable/SignatureHelp/AbstractCSharpSignatureHelpProvider.cs

```diff
@@ -53,7 +56,7 @@ protected static SymbolDisplayPart NewLine()
                 parameter.Name,
                 parameter.IsOptional,
                 parameter.GetDocumentationPartsFactory(semanticModel, position, formatter),
-                parameter.ToMinimalDisplayParts(semanticModel, position));
+                parameter.ToMinimalDisplayParts(semanticModel, position, s_allowDefaultLiteralFormat));
```

Comment:
Does this need to take into account language version, so we don't show default literals to people on C# <7.1? Or do we not worry about that sort of thing? Or does ToMinimalDisplayParts take care of that?

Reply:
@davidwengier I don't know if ToMinimalDisplayParts accounts for language version or not.
Probably @sharwell or @CyrusNajmabadi know?

Reply:
It doesn't need to account for lang version.  How we present info is different from what suggestions we offer, or what code we generate :)